website/app/styles/pages/application/sidebar.scss

```diff
@@ -52,17 +52,18 @@ body.application.index {
   }
 }
 
-// ERROR PAGE
+// PAGES THAT DO NOT WANT SIDEBARS
```

Comment:
@hashicorp/hds-engineering I think this is the suggested approach but if you look through this commit, you'll see that there are more than a few places where we have to add the new class for any page where we don't want a sidebar. 

The other issue is that the sidebar code is still in the DOM. 

I don't mind it if we feel like we should ship it via hiding in CSS instead of conditional statements in our JS, but it's not the recommended approach.

Reply:
@MelSumner @KristinLBradley 
- adding more and more classes to the `not()` selection was is not the solution. When the logic for the sidebar was built, was never intended to be used this way. Please, let's not start adding more classes there. 
- If the sidebar should be visible or not, should be controlled via JS and conditionals _IF_ it's necessary. At the moment we have only two use cases:
  - the sidebar in the "patterns" page has already been discussed: we know is a temporary case, and soon (probably even before the launch) will have at least one item, so we can leave it this way. it's not perfect, but it's OK
  - the sidebar in the "error" page is needed, we can't just hide it from the DOM (so my suggestion yesterday is also not entirely correct, needs to be tweaked) because on small screens the sidebar contains the main navigation

As asked yesterday, please remove all the extra code and keep the solution simple. We're spending too much time and energy on a problem that can be solved with a few simple lines of CSS:
```
body.application.error {
  @include breakpoint.with-fixed-sidebar () {
    .doc-page-sidebar {
      display: none;
    }
  }
}
```

Reply:
@MelSumner can we get this suggestion applied so we can get this change wrapped up?